packages/sui-test/package.json

```
@@ -21,6 +21,7 @@
     "@babel/polyfill": "7.12.1",
     "@babel/register": "7.13.0",
     "@s-ui/helpers": "1",
+    "@testing-library/cypress": "^7.0.4",
```

Should version be fixed?

```suggestion
    "@testing-library/cypress": "7.0.4",
```